README.rst

```diff
@@ -167,15 +184,18 @@ We follow the `all-contributors specification <https://github.com/alan-turing-in
 
 If you have a question, `chat <https://gitter.im/sktime/community?source=orgpage>`__ with us or `raise an issue <https://github.com/alan-turing-institute/sktime/issues/new/choose>`__. Your help and feedback is extremely welcome!
 
+All new contributors are welcome. You can join to contribute by:
```

Comment:
This section now reads a bit strange, I like your list of entry points below for how to get involved, perhaps organising them in a table would be good. 

For example:
| How to get involed | |
|---|---|
| [Mentoring](...) | Do you want to contribute to sktime but need some help? |
| [Meet-ups](...) | ... |